### DISCUSSION TITLE
I can't uninstall ImageMagick

### DISCUSSION THREAD

**Discussion by DCZurbano71** [ImageMagick/ImageMagick#4295] [Consulting]
I installed Ubuntu 20.04 and by default it came installed as ImageMagick application. Since I don't use this type of application, I want to uninstall it and I can't. Neither from the Ubuntu Software panel nor by Terminal. I hope you can help me. Thanks

**Comment by urban-warrior** [comment]
Have you tried https://www.google.com/search?q=ubuntu+uninstall+imagemagick?  You may not use the ImageMagick application but there are plenty of systems apps that do depend on it-- meaning you may break other apps that you do need.
